### ISSUE TITLE
Make Heading 2 the top-level heading for chapter/part content

### ISSUE THREAD

**Issue by josieg** [pressbooks/ideas#409]
### Feature Description
In the Pressbooks webbook, chapter titles are always set to Heading 1. In addition, the top-level heading in the content editor is Heading 1 (and this is the heading writers are encouraged to start with, especially if they want to use the two-level table of contents or collapse heading features).

However, to leads to inaccurate heading structure, with the chapter title and main headings of the chapter all being on the same heading level. 

This should be changed so Heading 2 is the top-level heading that an author can select in their chapter/part. This would also mean that the two-level table of contents and the collapse heading feature would apply to Heading 2s rather than heading 1s.

### Feature Use Case
This change is important for accessibility. In particular, for readers who navigate content with screen readers. Screen readers will communicate the heading levels so someone who cannot see the page can understand how topics relate to each other. With both chapter titles and main topic headings set to heading 1, it is unclear that the first heading (the chapter title) is the title for all content on that page.

This change will also ensure content creators don't have to choose between using proper heading structure or the features tied to Heading 1s.

### Other Notes

![PB heading structure](https://user-images.githubusercontent.com/23706762/153456599-ff724ede-08a8-4588-aa09-2bd389395dc0.jpg)
This picture shows the current heading structure in the [Images chapter in the Accessibility Toolkit](https://opentextbc.ca/accessibilitytoolkit/chapter/images/). Right now, the chapter title and main headings are all heading 1. However, the main headings and subheadings should all be bumped down a heading level.

Related issue: https://github.com/pressbooks/ideas/issues/221

**Comment by SteelWagstaff** [comment]
Thanks @josieg -- we've discussed this very issue/problem internally. We had hoped that browsers and other reading tools would have supported H1s inside of section elements differently, but the reality is that they haven't and probably never will (see https://html5accessibility.com/stuff/2021/03/21/a-decade-a-year-of-heading-backwards/ and https://html5doctor.com/computer-says-no-to-html5-document-outline/), so in practice I think you're exactly right. This will be pretty difficult to implement since it would have knock-on effects for so many users/books, etc., but it's on our mind and hopefully we'll be able to do something along these lines before too long.

**Comment by josieg** [comment]
Thanks Steel! I figured it was on your radar. Even just making so people could choose to have the collapse headings/two-level TOC apply to either H1 or H2 would make it possible to use H2 for new books being created.

**Comment by SteelWagstaff** [comment]
Still relevant. Have discussed with Ned a few times. Would be pretty massive undertaking to shift all H1s to H2s, all H2s to H3s, etc. If we did that in the content, we could then improve the way that 2 part organization, etc. works in Pressbooks.

**Comment by MichelleWeremczuk** [comment]
This change is important for accessibility. From my understanding, it should also be prioritized before implementing any upcoming navigation changes to the reading interface.

**Comment by SteelWagstaff** [comment]
An ideal solution: build a tool that inspects content elements in a book or database. If it finds H1 tags within the content block, shift all H5 to H6, all H4 to H5, all H3 to H4, all H2 to H3, and all H1 to H2. Should also warn the user if it had both H1 and H6 elements (since existing H6 elements cannot be down shifted further). We would also need to address how themes handle styling of these heading elements (H1 currently have styles defined -- they should probably be shifted to apply to H2, etc to avoid disrupting the appearance of current books).

**Comment by arzola** [comment]
This issue has been automatically closed as part of an organization-wide backlog review.

**Reason:** This issue was created 4y ago with no updates in the last 6m.

If this is still relevant, please feel free to reopen it with updated context. We appreciate your contributions and want to ensure our issue tracker stays actionable.

**Learn more:** https://pressbooks.community/t/announcement-automatic-closure-of-old-github-tickets/4542

**Comment by greatislander** [comment]
This is a priority accessibility issue.
